### PR TITLE
kata-manager: do not depend on git,go or local repos

### PR DESCRIPTION
**Pull request by marcov**
Make kata-manager able to run on base non development systems,
where git, golang and kata local repos may not be available.

Fixes: #828 

Signed-off-by: Marco Vedovati <mvedovati@suse.com>

### CODE REVIEW

File: cmd/kata-manager/kata-manager.sh [hunk 1]
```diff
@@ -10,28 +10,38 @@ set -e
...
+typeset -r tarball_suffix="/archive/master.tar.gz"
 
 typeset -r config_file_name="configuration.toml"
 typeset -r default_image_dir="/usr/share/kata-containers"
 typeset -r default_image_file="${default_image_dir}/kata-containers.img"
 typeset -r default_initrd_file="${default_image_dir}/kata-containers-initrd.img"
 typeset -r default_config_file="/usr/share/defaults/kata-containers/${config_file_name}"
 typeset -r local_config_file="/etc/kata-containers/${config_file_name}"
+typeset -r kata_doc_to_script="${test_repo}/.ci/kata-doc-to-script.sh"
+typeset -r -A dowloaders_list=([curl]="-fsSL" [wget]="-O -")
```
Comment (jodh-intel) [inline cmd/kata-manager/kata-manager.sh]: cute! :)

Comment (jodh-intel) [inline cmd/kata-manager/kata-manager.sh]: cute! :)

Nit: Minor typo though as this should probably be `downloaders_list` (missing `n`)?

Comment (jodh-intel) [inline cmd/kata-manager/kata-manager.sh]: Cute!

Nit: Typo as this should be `downloaders_list` presumably (missing `n`)?

Comment (jodh-intel) [inline cmd/kata-manager/kata-manager.sh]: Cute!

Nit: Typo as this should be `downloaders_list` presumably (missing `n`)?

Comment (jodh-intel) [inline cmd/kata-manager/kata-manager.sh]: Cute!

Nit: Typo as this should be `downloaders_list` presumably (missing `n`)?

Comment (jodh-intel) [inline cmd/kata-manager/kata-manager.sh]: Cute!

Nit: Typo as this should be `downloaders_list` presumably (missing `n`)?

Comment (jodh-intel) [inline cmd/kata-manager/kata-manager.sh]: Cute!

Nit: Typo as this should be `downloaders_list` presumably (missing `n`)?

Comment (jodh-intel) [inline cmd/kata-manager/kata-manager.sh]: Cute!

Nit: Typo as this should be `downloaders_list` presumably (missing `n`)?

**Comment by marcov** [comment]
Glad to see this works for you too :) 

I guess I can remove the WIP then,, and thanks for spotting the typo.

**Comment by jodh-intel** [comment]
Umm, not sure what happened here. I close the issue as I thought this PR was merged but github tells me I closed it?!?

I'm going to merge this with the following justification:

- We *must* have a working installation method for new users.
- @marcov, @chavafg and myself have tested this locally.
- The `kata-manager` is only used by users and osbuilder so there is little point letting the CI chug through the PR as the change is effectively a NOP as far as this and the other code repos are concerned.

If there are issues, we can fixup in a follow-up PR. I'm about to re-test once it lands too and will report back here...

**Comment by jodh-intel** [comment]
Tested latest `kata-manager` using https://github.com/kata-containers/documentation/blob/master/install/installing-with-kata-manager.md on 16.04 and works a treat.

Hopefully, we'll get the automated CI tests on https://github.com/kata-containers/documentation/pull/280 landed "real soon now" :)

**Pull request merged**